lhcb-YYYY-open-data-curation/LICENCE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
We can remove LINCENSE from this directory, since we have the top-level one already.

Reply:
Sure, sounds good.